src/Microsoft.AspNetCore.Server.Kestrel/Http/MessageBody.cs

```diff
@@ -178,6 +178,7 @@ public override ValueTask<int> ReadAsyncImplementation(ArraySegment<byte> buffer
         private class ForChunkedEncoding : MessageBody
         {
             private int _inputLength;
+            private int _chunkSize;
```

Comment:
was important when `ChunkPrefix` -> `ChunkSuffix` to consume the trailer to detect the type after a return from `ChunkData`, isn't important now - will revert.